src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -450,24 +450,21 @@ message BackplaneStatus {
 
   DispatchedOperationsStatus dispatched_operations = 9;
 
-  // Maintained for backward compatibility.
-  repeated string active_workers = 4;
+  // Maintained for backward compatibility. Union of active_storage_workers and active_execute_workers
+  repeated string active_workers = 4 [deprecated = true];
 
-  repeated string active_storage_workers = 12;
-
-  repeated string active_execute_workers = 13;
-
-  int64 cas_lookup_size = 5;
-
-  int64 action_cache_size = 6;
+  reserved 5;
+  reserved 6;
+  reserved 7;
+  reserved 8;
+  reserved 10;
```

Comment:
You can reserve a range of field sequence numbers: https://github.com/bazelbuild/remote-apis/blob/main/build/bazel/remote/execution/v2/remote_execution.proto#L608, and let's get a comment in there